docs/about-lightning.md

```diff
@@ -0,0 +1,228 @@
+- Add custom logic to handle a step failure (fail triggers)
+
+First of class monitoring:
+
+- Get notified on run failures
+- View the status of every run
+- Search workflow runs by input/output data and logs
+- Filter workflow runs by status, workflow name and date
+
+Re-processing functionality:
+
+- Bulk reprocess requests to address any issues or process requests with updated
+  logic
+
+### Collaboration is key
+
+On one hand, the users that understand what processes need automating are (more
+often than not) business analysts, not developers. They’re the experts on what
+needs to happen when and where, and they’re very capable of planning out
+integrations and putting together mapping specifications and bpmn flows.
+
+On the other hand, integrations often require custom logic that cannot be
+simplified through low-code and therefore must be implemented by software
+engineers.
+
+That’s why **the best integrations are built when non-technical users and
+developers collaborate**. Lightning is being developed to bridge the gap between
+non-technical and technical users through:
+
+Intuitive, user-friendly user interface for non-technical users:
+
+- Understand a workflow in a visual, human-readable format (abstract away from
+  code to make workflows understandable to non-technical users)
+- Build credentials through a form (remove the need to read through confusing
+  API documentation)
+- Build requests through a form
+- Save mappings used in workflows as constants so they can be easily viewed and
+  edited without needing to read code
+- Clear documentation for users to learn how to plan and build integrations
+
+Projects-as-code and CLI for a developer interface:
+
+- Export, import and configure projects as code in the code editor of your
+  choice
+- Run, test and deploy projects through a command line interface
+- Review and track changes through github version control
+
+Collaboration functionality:
+
+- Track changes through version control
+- Rollback to a previous version
+- Get notified when a workflow is changed
+- Share a link to a specific workflow error on the runs history page
+- Share a link to a specific workflow step within the builder
+- Add collaborators as view-only user or editor to a project
+- Audit all changes made to credentials
+
+### It’s not ‘just’ a request or a piece of data, it’s a person
+
+OpenFn specialises in integration tooling for the health and humanitarian
+sector. This means that behind every piece of data which comes in through a
+request lies a person in need of critical services. This is why Lightning
+focusses on:
+
+Accountability:
+
+- Credential audit trail
+- Version control
+
+Security:
+
+- Secure credential management (encrypted at REST, credential secrets are
+  scrubbed from logs, secure credential sharing
+- Zero-persistence pipelines
+- Role-based project access
+- Additional authentication rules for webhooks
+
+### Standards and compliance matter
```

Comment:
Standards is a major selling point... and this is the first time we mention "Digital Public Good"! I think this should maybe be moved to the top.